src/config/dynamic-theme-fixes.config

```diff
@@ -1501,6 +1501,12 @@ code:first-of-type {
     box-shadow: 0 0 4px ${rgba(3,9,15,0.15)} !important;
     background-color: ${rgba(3, 9, 15, 0.02)} !important;
 }
+#ak-main-content {
+    background-color: rgb(22, 23, 25) !important;
+}
```

Comment:
I think instead of this, you can use something like this, or a variation of it:
```css
#jira-frontend {
  background: var(--ds-surface, transparent) !important;
}
```

This way it will work with custom set colours in DR; e.g., yours sets the exact dark colour to the area, while mine overrides the white on element that's actually causing it, to transparent. (the variable is there just because it's like that originally, just with `white` instead of `transparent`)

Reply:
Looks like my PR was merged so by all means please raise a separate follow-up PR. I just did what worked for my use case.